### PR TITLE
add x86 mnemonics and alias to Jz

### PR DESCRIPTION
**Pull request by srounet**
Add some x86 Instructions to simple_x86 along with some basic testing for coverage.

- Popal, Popad, Popfd
- Pushal, Pushad, Pushfd
- Je

### CODE REVIEW

**Comment by hakril** [comment]
Hi,

Thank you for the PR.
Could you please also add the Pushfd, Popfd & Jz instructions to the simple_x64 file please ?
I would like to keep the instruction set available to both bitness somewhat similar.
If am not mistaken Pushad/Popad do not exist in x64 so we can ignore these in simple_x64
Finally I prefer to avoid the Popal/Pushal aliases as no reference to it can be found in the intel manual (I used https://www.intel.com/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-instruction-set-reference-manual-325383.pdf) as a reference.

**Comment by srounet** [comment]
Thank you for your feedback as discussed I remove the pushal/popal from x86 and used `expected_result` within tests.
Pushfd, Popfd and Jz have also been added to x64 instructions.

Feel free to suggest any changes.

File: tests/test_simple_x64.py [hunk 1]
```diff
@@ -137,6 +137,9 @@ def test_assembler():
     CheckInstr(Add)('RAX', mem('[0x7fffffff]'))
     CheckInstr(Add)('RAX', -1)
 
+    # Registers
```
Comment (hakril) [inline tests/test_simple_x64.py:140]: Looks like the x64 instruction should be Pushfq & Popfq 
The DWORD equivalent cannot  be encoded in 64 bits mode but The intel manual match de capstone result for this one (present in manual intel : PUSHF/PUSHFD—Push EFLAGS Register onto the Stack)

File: windows/native_exec/simple_x64.py [hunk 2]
```diff
@@ -1011,6 +1014,14 @@ class Mov(Instruction):
                 (RawBits.from_int(5, 0xb8 >> 3), X64RegisterSelector(), Imm64())]
 
 
+class Pushfd(Instruction):
```
Comment (hakril) [inline windows/native_exec/simple_x64.py:1017]: Looks like the x64 instruction should be Pushfq.
The DWORD equivalent cannot be encoded in 64 bits mode.

File: windows/native_exec/simple_x64.py [hunk 3]
```diff
@@ -1011,6 +1014,14 @@ class Mov(Instruction):
...
+    encoding = [(RawBits.from_int(8, 0x9c),)]
+
+
+class Popfd(Instruction):
```
Comment (hakril) [inline windows/native_exec/simple_x64.py:1021]: Looks like the x64 instruction should be Popfq.
The DWORD equivalent cannot be encoded in 64 bits mode.

**Review by hakril — COMMENTED** [review]
Almost good for me.
Just need to adjust the PUSHFD name in x64 which is PUSHFQ according to the intel manual (and the capstone result apparently)

**Comment by hakril** [comment]
Thank you for the PR and the following fixes.
Do not hesitate if you have any other issues or proposition for improvement

**Pull request merged**